build-system/pr-check/remote-tests.js

```diff
@@ -82,7 +82,12 @@ async function main() {
       buildTargets.has('FLAG_CONFIG') ||
       buildTargets.has('INTEGRATION_TEST')
     ) {
-      timedExecOrDie('gulp integration --nobuild --compiled --saucelabs');
+      timedExecOrDie(
+        'gulp integration --nobuild --compiled --saucelabs --stable'
+      );
+      timedExecOrDie(
```

Comment:
I wonder if this should be changed to `timedExec` (without `orDie`) so beta tests cannot fail Travis irrespective of what kind of error is encountered. For example, should the "karma detected zero tests" error fail beta browser tests? WDYT?

Reply:
Good idea, trying that.